backend/src/libs/packages/server-application/libs/constants/api.constants.ts

```diff
@@ -1,9 +1,12 @@
 import { APIPath } from '#libs/enums/enums.js';
 import { AuthApiPath } from '#packages/auth/auth.js';
 
+const SERVED_PAGE_PATH = '/*';
+
 const WHITE_ROUTES = [
   { path: `/api/v1${APIPath.AUTH}${AuthApiPath.SIGN_UP}`, methods: ['POST'] },
   { path: `/api/v1${APIPath.AUTH}${AuthApiPath.SIGN_IN}`, methods: ['POST'] },
```

Comment:
Let's do this way:
1. Add /documentation to APIPath enum
2. In initMiddlewares replace routePrefix for swaggerUi with `/api/${api.version}/${APIPath.DOCUMENTATION}`
After that we should be able to access it from (this step will make it accessible from frontend by /api/v1/documentation route)
3. in WHITE_ROUTES wrap paths to new RegExp. Don't forget to add ^ - start,  $ - end, so that we match exact routes for sign_up / sign_in
4. for SERVED_PAGE_PATH add \ before *, so that we match exact route, not any path
5. Add new white-route - ```new RegExp(`/api/v1${APIPath.DOCUMENTATION}/*`)```
6. in checkIsWhiteRoute helper use test instead of explicit comparison

After this it should be accessible from backend as well ✨ 

Note: don't forget to test RegExps that they match correct routes

Reply:
can't we simply change `request.routerPath` to `request.url`? With this change, it understands the `api/v1/documentaion` route

Reply:
do you have other response then `not found` without the authorization plugin?

Reply:
we will still need to allow served pages for the deploy because backend will need to return static pages

Reply:
but with this fix it is allowed

Reply:
now you can access all unregistered routes

Reply:
you can disconnect the authorization plugin and you'll see, that nothing actually changed. The documentation route will not be found
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/58943798/d5aff9a4-2c63-4363-8f48-822bc972c0a2)

Reply:
just checked, this won't work if we will return static pages from backend (needed for deploy). Because instead of /* it will return /sign-in for example and we won't be able to detect that it's static except if we check that route starts with /api/v1, but this approach is not good

Reply:
so I have to do the suggested above method anyway, right?

Reply:
yep)

Reply:
ok, no problem then)

Reply:
@Liza-Veis, could you help me understand, what you mean by `Because instead of /* it will return /sign-in for example and we won't be able to detect that it's static`? When I tried to access a `sign-in` route it showed me `/*`. Or there would be another response in production?
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/58943798/91c478d4-3907-4765-a7ae-b4c7dccb6fe3)

Reply:
Simple steps to reproduce how deploy will work:
1. stage your changes `!important`
2. run `npm run build -w frontend`
3. copy frontend build folder to backend folder and rename it to public
4. run  `npm run build -w backend`
5. go to `localhost:3001/sign-in`
6. it should return sign-in page

Reply:
UPD: even with your steps above, you can see that request.url returns /sign-up instead of /*, so if we change request.routerPath to request.url, it won't work

Reply:
Should something be wrong while reproducing? Because I am receiving the `sign-in` page on frontend

Reply:
ok, I'll just make the suggested changes, cause I think we can discuss for a long time)

Reply:
hmm that's strange because I'm getting /sign-up for request.url

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/58943798/436bfc19-f023-4a61-ae1e-394613201042)

Reply:
sorry for confusing, I am getting this in request.routePath

Reply:
the only problem is that I am getting a not found from frontend part, when trying to access the `/api/v1/documentation` route

Reply:
I will try your suggested way and we'll see